docs/src/pages/customization/css-in-js.md

```diff
@@ -184,11 +184,11 @@ For instance, it can be used to defined a `getInitialProps()` static method (nex
 It will be linked to the component.
 Use the function signature if you need to have access to the theme. It's provided as the first argument.
 2. `options` (*Object* [optional]):
-  - `options.withTheme` (Boolean [optional]): Provide the `theme` object to the component as a property. It's `false` by default.
-  - `options.name` (*String* [optional]): The name of the style sheet. Useful for debugging.
+  - `options.withTheme` (Boolean [optional]): Default to `false`. Provide the `theme` object to the component as a property.
+  - `options.name` (*String* [optional]): Default to `null`. The name of the style sheet. Useful for debugging.
     If the value isn't provided, we will try to fallback to the name of the component.
```

Comment:
I have been removing personal pronouns from the docs (except where it is in reference to the team, contributors, the user etc.), so:

"If the value isn't provided, it will try to fallback to the name of the component."